pkg/cmd/cli/cmd/process.go

```diff
@@ -122,7 +127,16 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 	// When storedTemplate is not empty, then we fetch the template from the
 	// server, otherwise we require to set the `-f` parameter.
 	if len(storedTemplate) > 0 {
-		templateObj, err := client.Templates(namespace).Get(storedTemplate)
+		parts := strings.Split(storedTemplate, "/")
+		switch len(parts) {
+		case 3:
+			sourceNamespace, storedTemplate = parts[0], parts[2]
+		case 2:
+			storedTemplate = parts[1]
+		case 1:
+			storedTemplate = parts[0]
```

Comment:
check middle section to make sure it is empty or "template" so we don't allow invalid things we'll later break